wormhole-connect/src/components/Modal.tsx

```diff
@@ -61,10 +61,16 @@ function Modal({ open, width, closable, children, onClose }: Props) {
   return (
     <Dialog
       open={open}
-      onClose={onClose}
+      onClose={(event, reason) => {
+        console.log(reason);
+        if (reason === 'backdropClick') {
+          onClose();
+        }
+      }}
       sx={{ borderRadius: 8 }}
       fullWidth
       fullScreen
+      onClick={onClose}
```

Comment:
onClose is fired on click?

Reply:
@anondev2323 confirmed, this causes the dialog to close even when you click on, like, the search box or the heading, etc.